mw-webapp/src/hooks/useConvertingMarkdownToUrl.tsx

```diff
@@ -0,0 +1,37 @@
+import {useMemo} from "react";
+
+/**
+ * The render Description hooks processes the description text to find markdown-style
+ * links and convert them into
+ * clickable anchor tags.
+ */
+export const useConvertingMarkdownToUrl = (description: string) => {
```

Comment:
we already have renderMarkdown

Reply:
Deleted the useRenderMarkdowmToUrl file, replaced it with the renderMarkdown function in the userCard and WayCard components. And as for the comments, I always seem to leave